tests/preprocess/test_transformer.py

```diff
@@ -8,9 +8,12 @@
 from numalogic.preprocess import LogTransformer, StaticPowerTransformer, TanhScaler
 
 
+RNG = np.random.default_rng(42)
```

Comment:
There are a couple of np.random related code refactors. Were they the result of linter warning?

Reply:
No, this was manually done. Linters threw an error. Using a random number generator is the preferred approach for numpy as per their docs.